Gopkg.toml

```diff
@@ -47,7 +47,7 @@ ignored = [
   name = "github.com/go-sql-driver/mysql"
   branch = "master"
 
-# https://github.com/grpc-ecosystem/grpc-gateway/commit/893772d
+# https://github.com/grpc-ecosystem/grpc-gateway/pull/497
```

Comment:
Slightly nicer if this comment references the commit (`https://github.com/grpc-ecosystem/grpc-gateway/commit/8db8c1a`) because it's easier to see from the github commit view when the commit has been included in a release.

Reply:
Ok will do